PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,34 +1,34 @@
-## Description of Work
-<!--add description of work here 🙏🏼 -->
+## Description
+<!--add a short description of work here 🙏🏼 -->
 
 ### Change List
-
-- [] item 1
-- [] item 2
-
-## Deployment Notes
-
-### Deployment Checklist
-
-`select all that apply`
-
-### Code changes
-
-- [] Unit/Integration tests added
-- [] README updated
-- [] `/docs` updated
-
-### Infra Changes
-
-- [] All environments updated
-- [] Applicable Lucidcharts updated
-- [] If modules were moved were resources cleaned up
-
-## Technical Debt
+<!-- More detailed list of the major changes that will help explain what is going on with your PR -->
+- item 1
+- item 2
+
+<details>
+<summary>Extra Stuff</summary>
+  <!-- select all that apply -->
+  <!-- More details from GitHub -->
+  <!-- https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax -->
+  <!-- https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting -->
+        
+  ### Diagrams
```

Comment:
@jimstrate 
Would it make sense to make this a Diagrams/Proof of work section? 
We talked about that in the past. How has it been working on your end, especially for FE PRs?
cc @TheManSpeaker

Reply:
yeah a screenshot section would be cool.  It works great for FE development.  Showing a curl command executions captures a lot too for BE if it's a new endpoint.

Reply:
I think with this as the default across all repos, it makes more sense to leave it out and just add it to that specific repo?

Reply:
I think for most repos it makes sense to have some sort of general "show your work" section, though I don't want to be too prescriptive as to what that looks like because even within a repo different types of changes might have different types of output i.e. terraform plans, curl commands, screenshots, etc

Reply:
and sometimes it's just maybe unit tests passing around a piece of functionality that's not actually exposed by a public interface yet 🤷

Reply:
Added a new section below